Live-Snap/Live-Snap/Models/ImageManager.swift

```diff
@@ -37,10 +37,7 @@ class ImageManager {
     }
     
     func photoLibraryPermissionWasDenied() {
-     
-        let deniedPermissionViewControllerIsActive: Bool = System.shared.appDelegate().pageViewController?.viewControllers![0] is DeniedPhotoLibraryPermissionViewController
-        
-        if !deniedPermissionViewControllerIsActive {
+        DispatchQueue.main.async {
```

Comment:
ImageManager is a model and should not control the view hierarchy. this code doesnt belong here